src/htcondor_es/convert_to_json.py

```diff
@@ -827,11 +830,18 @@ def convert_to_json(ad, cms=True, return_dict=False, reduce_data=False):
     #
     # We will use the postjob_status_decode dict to decode
     # the status. If there is an unknown value it will set to it.
+    # Note: if the completed task has not a committed time 
```

Comment:
do you mean that the jobs we were looking at yesterday do not have a Committed Time from condor ? Do we understand why ?

Reply:
Because they were removed, marked as finished by the post-job.